### ISSUE TITLE
Commands not running when pause deactivated

### ISSUE THREAD

**Issue by edgary777** [foreshadow/atom-python-run#52]
Had been using the plugin for a while without a problem, but today the commands stopped working when Pause is deactivated.
No error message is thrown but sometimes the mouse cursor shows activity after pressing the command keys.
It works perfectly when Pause is activated
I'm using Windows 10

**Comment by xovertheyearsx** [comment]
If you `Ctrl + Alt + I` you'll see something like this.

```
atom-python-run: `pause` and `pipe` options have been disabled.
Cowardly refusing to continue...
```

`terminal` opens a process where `cp` becomes a child process of the `shell` process. There is no (known) way to effectively communicate with that process. `cp` is used to communicate with that process in advance. Pause tells `cp` to keep the console window open. All commands are passed to `cp` which also doubles as a filter. This filter logs any executed commands.

_Executing without pausing, or writing to a file, sounds like a desirable feature request._ 

I never coded it as an option even though I thought about this in advance. I locked it by default because `cp` just overwrites its previous logs instead of appending.

I locked it so to ensure no unintended commands slipped by. It would refuse to execute unless certain conditions were met. _The first condition is that it pauses_. _The second condition is that if it doesn't pause, it should write to a file instead of the terminals standard output method_.

So in all, this isn't a bug. It's doing exactly what it's supposed to. That way, ideally, you execute only what you intended to.

In a future revision, @foreshadow or my self, could slightly tweak the code so that's it more flexible. For now, I apologize for any inconvenience.
